test/e2e/tests/metrics/state-snapshots/errors-after-init-opt-in-background-state.json

```diff
@@ -256,18 +256,24 @@
     "lostIdentities": "object",
     "forgottenPassword": false,
     "preferences": {
+      "dismissSmartAccountSuggestionEnabled": "boolean",
+      "featureNotificationsEnabled": "boolean",
       "hideZeroBalanceTokens": false,
+      "petnamesEnabled": "boolean",
+      "privacyMode": "boolean",
       "showExtensionInFullSizeView": false,
       "showFiatInTestnets": false,
       "showTestNetworks": false,
+      "skipDeepLinkInterstitial": "boolean",
```

Comment:
I don't think this should be in this PR

Reply:
These fields were missing because previously the shallow merge lost them